python/lsst/ts/ATWhiteLightSource/chillerComponent.py

```diff
@@ -35,7 +36,10 @@ def __init__(self):
         #    print(e)
         #    self.log.exception(e)
         
-        self.writer.close()
+        try:
```

Comment:
I think it would be better if you check that `self.writer is not None` before calling `close`, instead of letting the exception do that work for you. 

something like;

```
if self.writer is not None:
     self.writer.close()
```